src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -109,6 +110,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Text.Data" Version="14.0.23205" />
     <PackageReference Include="Microsoft.VisualStudio.Text.Logic" Version="14.0.23205" />
     <PackageReference Include="Microsoft.VisualStudio.Text.UI" Version="14.0.23205" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="14.0.50702" />
```

Comment:
This package was not directly imported before and instead was taken from Shell. If I don't directly import it, it will be taken from Shell 15, so I preferred to explicitly import the 14 version.